index.bs

```diff
@@ -690,49 +690,46 @@ partial interface MLContext {
-        <div class=validusage>
-            1. For each |key| -> |value| of |inputs|:
-                1. |graph|.{{MLGraph/[[inputDescriptors]]}}[|key|] must [=map/exist=].
-                1. Let |inputDesc| be |graph|.{{MLGraph/[[inputDescriptors]]}}[|key|].
-                1. The type of {{ArrayBufferView}} |value| must match |inputDesc|.{{MLOperandDescriptor/type}} according to [this table](#appendices-mloperandtype-arraybufferview-compatibility).
-                1. |value|.\[[ByteLength]] must equal to [=byte length=] of |inputDesc|.
-            1. For each |key| -> |value| of |outputs|:
-                1. |graph|.{{MLGraph/[[outputDescriptors]]}}[|key|] must [=map/exist=].
-                1. Let |outputDesc| be |graph|.{{MLGraph/[[outputDescriptors]]}}[|key|].
-                1. The type of {{ArrayBufferView}} |value| must match |outputDesc|.{{MLOperandDescriptor/type}} according to [this table](#appendices-mloperandtype-arraybufferview-compatibility).
-                1. |value|.\[[ByteLength]] must equal to [=byte length=] of |outputDesc|.
-        </div>
-    1. For each |key| -> |value| of |inputs|:
-        1. Let |inputDesc| be |graph|.{{MLGraph/[[inputDescriptors]]}}[|key|].
-        1. Let |inputTensor| be a new tensor for |graph|.{{MLGraph/[[implementation]]}}.
-        1. Set the data type of |inputTensor| to the one that matches the [=element type=] of {{ArrayBufferView}} |value|.
-        1. Set the dimensions of |inputTensor| to |inputDesc|.{{MLOperandDescriptor/dimensions}}.
-        1. Set the values of |inputTensor| to the values of |value|.
-        1. Set the input of |graph|.{{MLGraph/[[implementation]]}} that is associated with |key| to |inputTensor|.
-    1. For each |key| -> |value| of |outputs|:
-        1. Issue a compute request for output of |graph|.{{MLGraph/[[implementation]]}} that is associated with |key|.
-        1. Wait for the compute request to be completed.
-        1. If there is an error returned by |graph|.{{MLGraph/[[implementation]]}}, then:
-            1. Throw an "{{OperationError}}" {{DOMException}} and stop.
-        1. Else:
-            1. Let |outputTensor| be the output tensor returned by |graph|.{{MLGraph/[[implementation]]}}.
-            1. If the data type of |outputTensor| doesn't match the [=element type=] of {{ArrayBufferView}} |value|, then throw a "{{DataError}}" {{DOMException}} and stop.
-            1. If the byte length of |outputTensor| is not equal to |value|.\[[ByteLength]], then:
-                1. Throw a "{{DataError}}" {{DOMException}} and stop.
-            1. Else:
-                1. Set the values of |value| to the values of |outputTensor|.
-    1. Return {{undefined}}.
 </div>
 
+The {{MLContext/computeSync(graph, inputs, outputs)}} method steps are:
+1. Invoke <a>execute graph</a> given |graph|, |inputs| and |outputs|.
+1. If that throws and error, re-throw the error and stop.
```

Comment:
s/and/an (fixed in the other place, missed this one - will be fixed with other possible bugs)

Reply:
fixed